examples/off_limb_enhance.py

```diff
@@ -52,25 +52,37 @@
 params = np.polyfit(rsun_array[rsun_array < 1.5],
                     np.log(y[rsun_array < 1.5]), 1)
 
+###############################################################################
+# Tell matplotlib to use LaTeX for all the text, make the fontsize bigger, and
+# then plot the data and the fit.
+
+fontsize = 14
+plt.rc('text', usetex=True)
+plt.rc('xtick', labelsize=fontsize)
+plt.rc('ytick', labelsize=fontsize)
 plt.plot(rsun_array, y, label='data')
-label = r'fit=Aexp({:.2f}r)'.format(params[0])
-plt.plot(rsun_array, np.exp(np.poly1d(params)(rsun_array)), label=label)
-plt.yscale('log')
-plt.ylabel('mean DN')
-plt.xlabel('radius r (Rsun)')
-plt.legend()
+best_fit = np.exp(np.poly1d(params)(rsun_array))
+label = r'best fit: {:.2f}$e^{{{:.2f}r}}$'.format(best_fit[0], params[0])
+plt.plot(rsun_array, best_fit, label=label)
+plt.yscale(r'log', fontsize=fontsize)
```

Comment:
why not `plt.rc` these fontsizes as well?

Reply:
I could have, but then it would have changed the fontsize in the map plots.  Maybe I should do that too.